jsondatetime/jsondatetime.py

```diff
@@ -1,3 +1,4 @@
+import six
```

Comment:
I'd rather not take a dependency for this. Can't we `import sys` and then test `sys.version_info.major`and `sys.version_info.minor`?